package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "devDependencies": {
     "babel-core": "^6.25.0",
+    "ember-modules-codemod": "^0.1.5",
```

Comment:
Can you move this to be a dependency?  As a dev dep, any consumers won't have it